### PR TITLE
bench results: show phys mem if known

### PR DESCRIPTION
**Pull request by bp0**

### CODE REVIEW

File: modules/benchmark/bench_results.c [hunk 1]
```diff
@@ -153,14 +162,17 @@ bench_machine *bench_machine_this() {
...
+        m->is_su_data = (getuid() == 0);
         m->board = module_call_method("devices::getMotherboard");
         m->cpu_name = module_call_method("devices::getProcessorName");
         m->cpu_desc = module_call_method("devices::getProcessorDesc");
         m->cpu_config = module_call_method("devices::getProcessorFrequencyDesc");
         m->gpu_desc = module_call_method("devices::getGPUList");
         m->ogl_renderer = module_call_method("computer::getOGLRenderer");
         tmp = module_call_method("computer::getMemoryTotal");
-        m->memory_kiB = atoi(tmp);
+        m->memory_kiB = strtoll(tmp, NULL, 10);
```
Comment (lpereira) [inline modules/benchmark/bench_results.c]: Shouldn't this be strtoull()?

File: modules/benchmark/bench_results.c [hunk 2]
```diff
@@ -371,7 +390,7 @@ char *bench_result_benchmarkconf_line(bench_result *b) {
     char *bv = bench_value_to_str(b->bvalue);
 
 #define prep_str(s) (s ? (char*)auto_free(gg_key_file_parse_string_as_value(s, '|')) : "")
-    char *ret = g_strdup_printf("%s=%s|%d|%s|%s|%s|%s|%d|%d|%d|%d|%s|%s|%d|%d\n",
+    char *ret = g_strdup_printf("%s=%s|%d|%s|%s|%s|%s|%"PRId64"|%d|%d|%d|%s|%s|%d|%d|%d|%"PRId64"|%s\n",
```
Comment (lpereira) [inline modules/benchmark/bench_results.c]: This is becoming unwieldy...

File: modules/benchmark/bench_results.c (hunk 1 above)
Comment (bp0) [reply modules/benchmark/bench_results.c]: Yes.

File: modules/benchmark/bench_results.c (hunk 2 above)
Comment (bp0) [reply modules/benchmark/bench_results.c]: A little, but it remains compatible with existing results.

**Pull request merged**